rest-service/manager_rest/es_storage_manager.py

```diff
@@ -50,6 +51,15 @@
     'refresh': True
 }
 
+logging.basicConfig(filename='/var/log/cloudify/es_manager.log',
+                    level=logging.DEBUG,
+                    format='%(asctime)s - %(name)s - %(message)s')
+logger = logging.getLogger(__name__)
+logging.getLogger('manager-rest').setLevel(logging.ERROR)
```

Comment:
What is the current level of these loggers?
Will users be able to change the level from ERROR without changing code?